.doc_gen/metadata/bedrock-agent_metadata.yaml

```diff
@@ -209,3 +211,226 @@ bedrock-agent_GettingStartedWithBedrockAgents:
   services:
     bedrock-agent: {CreateAgent, CreateAgentActionGroup, CreateAgentAlias, DeleteAgent, DeleteAgentAlias, GetAgent, ListAgentActionGroups,
                     ListAgents, ListAgentKnowledgeBases, PrepareAgent}
+    
+
+
+
+bedrock-agent_GettingStartedWithBedrockFlows:
+  title: An end-to-end example showing how to create and invoke an Amazon Bedrock flow using an &AWS; SDK. 
+  title_abbrev: Create and invoke a flow
+  synopsis_list:
+    - Create an execution role for the flow.
+    - Create the flow.
+    - Deploy the fully configured flow.
+    - Invoke the flow with user-provided prompts.
+    - Delete all created resources.
+  category: Scenarios
+  languages:
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/bedrock-agent
+          excerpts:
+            - description: Generates a music playlist based on user-specified genre and number of songs.
+              snippet_tags:
+                - python.example_code.bedrock-agent-runtime.Scenario_GettingStartedBedrockFlows
+                - python.example_code.bedrock-agent-runtime.flow_invoke_flow
+                - python.example_code.bedrock-agent-runtime.run_playlist_flow
+                - python.example_code.bedrock-agent-runtime.Scenario_GettingStartedBedrockFlows_iam_role
+
+  services:
+    bedrock-agent: {CreateFlow, CreateFlowAlias, CreateFlowVersion, DeleteFlow, DeleteFlowVersion, DeleteFlowAlias, GetFlow, GetFlowAlias,
```

Comment:
The metadata seems to indicate several actions that I don't see executed in the code. For example, does the scenario use all of those Get and List actions?

Reply:
Removed metadata for list operations.